app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
* Kindly add the necessary actions in your controllers (Note: actions can be empty for now ) as stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=An%20action%20in%20the%20correct%20controller.)